pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -93,11 +98,13 @@ func TestCMServerDefaults(t *testing.T) {
 			LookupCacheSizeForRC:              4096,
 			LookupCacheSizeForRS:              4096,
 			LookupCacheSizeForDaemonSet:       1024,
+			ConfigureCloudRoutes:              true,
```

Comment:
I don't know what this does.  Do you know or have you disabled it?  Again, if not, disable it in our config and open an issue to figure out what its doing.